.github/CODEOWNERS

```diff
@@ -322,6 +358,8 @@
 # PRLabel: %Cognitive - Anomaly Detector
 /sdk/anomalydetector/                                                @conhua @mengaims @juaduan @moreOver0
 
+# AzureSDKOwners: @kristapratico
```

Comment:
Same question about service ownership.   I think the service owns this, so we'll need to remove `AzureSDKOwners` here so that the service owners defined below are considered the issue owners.